admin-dev/themes/new-theme/js/pages/product/product-map.ts

```diff
@@ -156,8 +156,12 @@ export default {
     bulkDeleteBtn: '#combination-bulk-delete-btn',
     bulkActionBtn: '.bulk-action-btn',
     bulkActionsDropdownBtn: '#combination-bulk-actions-btn',
-    bulkSelectAll: `#${bulkCombinationSelectAllId}`,
-    bulkSelectAllId: bulkCombinationSelectAllId,
+    //@todo: rename? its the checkbox that displays status of bulk selection (if one of options is checked: all or all in page)
+    bulkSelectAllDisplay: '#bulk-select-all-display',
```

Comment:
```suggestion
    bulkCheckboxesSwitch: '#bulk-checboxes-switch',
```
Not 100% sure of this one 😅. Just one suggestion but we can probably find better

Reply:
```suggestion
    bulkCheckboxesStatus: '#bulk-checkboxes-status',
```

Reply:
```suggestion
    selectAllCheckboxes: '#select-all-checkboxes',
```

Reply:
Im also in search of better naming, but none of your suggestions seems right :smile: 
I want to point out that its only for display (it cannot even be checked by user and is not used to detect if all are checked, because js overrides the behavior and uses the inputs in dropdown for that)

Reply:
I think Im going with `bulkAllPreviewInput`